src/metabase/dashboards/api.clj

```diff
@@ -972,7 +974,7 @@
    [:show_in_getting_started {:optional true} [:maybe :boolean]]
    [:enable_embedding        {:optional true} [:maybe :boolean]]
    [:embedding_params        {:optional true} [:maybe ms/EmbeddingParams]]
-   [:parameters              {:optional true} [:maybe [:sequential ms/Parameter]]]
+   [:parameters              {:optional true} [:maybe [:sequential ::parameters.schema/parameter]]]
```

Comment:
Using the registry is preferrable IMO because if we change the schema things will pick up the changes when they resolve it. If it's just a normal `def` things don't pick up the changes until you reload the namespace using that `def`.